Baikal/PostEffects/ML/inference_impl.cpp

```diff
@@ -0,0 +1,40 @@
+#include "inference_impl.h"
+
+#include <functional>
+#include <numeric>
+
+
+namespace Baikal
+{
+    namespace PostEffects
+    {
+        InferenceImpl::InferenceImpl(size_t width, size_t height) :
+            m_width(width), m_height(height)
+        {
+        }
+
+        Buffer InferenceImpl::GetInputBuffer()
+        {
+            return AllocBuffer(7);
+        }
+
+        void InferenceImpl::PushInput(Buffer&& buffer)
+        {
+            assert(buffer.shape()[0] == m_height);
+            assert(buffer.shape()[1] == m_width);
+            assert(buffer.shape()[2] == 7);
```

Comment:
I don't like seven, maybe make it const variable or make it configurable parametr